wordpress-mobile/WordPress-iOS
# ULS Google: Update Auth to show updated Google Signup confirmation view

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/282
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/384

This updates Auth to show the new Google Signup Confirmation view. It also enables the `unifiedGoogle` feature flag in development.

To test:
- `Continue with Google` from either login or signup.
- Select a Google account that has not been used to create a WP account.
- Complete Google auth.
- Verify the new Signup Confirmation view is displayed.
- Tap `Create Account`.
- Verify a new account is created.

<kbd>![confirmation](https://user-images.githubusercontent.com/1816888/90452838-55668400-e0ac-11ea-8fee-3d1e7cd51b2f.png)</kbd>

PR submission checklist:
